addons/point_of_sale/static/src/js/models.js

```diff
@@ -3626,4 +3631,15 @@ export class Order extends PosModel {
             return false;
         }
     }
+    /**
+     * Returns a random 5 digits alphanumeric code
+     * @returns {string}
+     */
+    _generate_ticket_code() {
+        var code = '';
```

Comment:
Prefer `let` over `var` or `const` if you don't intend to reassign the variable
```suggestion
        let code = '';
```